src/tests/toml-static-mod/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "toml-static"
+
+[package.metadata.system-deps]
+testdata = "4"
+teststaticlib = { version = "1", feature = "test-feature", modifiers = "+whole-archive"}
```

Comment:
I'm not familiar with `+whole-archive` and modifiers in general. If they are used only during static linking maybe they should be called `static_modifiers` instead?

What other modifiers could be used there for example?

Reply:
Thank you so much for taking a look so quickly and for pulling in some more people, I wouldn't consider myself a deep expert either! 

In [rustc documentation](https://doc.rust-lang.org/rustc/command-line-arguments.html#-l-link-the-generated-crate-to-a-native-library) the three specified are `[+|-]` `whole-archive`, `verbatim`, and `bundle`. This is where I wouldn't consider myself a deep expert and I'm not sure if more are supported and these 3 are just the documented ones because they have defaults.

The terminology I used here matches `rustc` and [build script](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-link-lib) documentation. However, I am happy to rename if that's the consensus.